test/e2e/deployimage/plugin_cluster_test.go

```diff
@@ -94,6 +94,7 @@ var _ = Describe("kubebuilder", func() {
 				"--plugins", "go/v3",
 				"--project-version", "3",
 				"--domain", kbc.Domain,
+				"--component-config=true",
```

Comment:
@laxmikantbpandhare 
Why we are adding it here?

Reply:
@camilamacedo86 - As explained [here](https://github.com/kubernetes-sigs/kubebuilder/pull/2826#issuecomment-1184974419), It is failing with `kustomize build`

Reply:
I re-ran after removal, you can see tests are failing.